components/utilities/updateAuctions.js

```diff
@@ -184,7 +184,6 @@ async function GetAuctionBlocks(api, currentBlock, startBlock, chain) {
     }
   } catch (error) {
     console.log(`Failure updating auction blocks on ${chain} at starting block ${startBlock}.`)
-    throw error.message;
```

Comment:
Error messaging/handling is actually more clear without this throw:
```
Failure updating auction blocks on Kusama at starting block 15724800.
```
as opposed to 
```
[UnhandledPromiseRejection: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason "Cannot read properties of null (reading '0')".] {
  code: 'ERR_UNHANDLED_REJECTION'
}
```